certora/README.md

```diff
@@ -0,0 +1,36 @@
+# Morpho token contract formal verification
+
+This folder contains the [CVL](https://docs.certora.com/en/latest/docs/cvl/index.html) specification and verification setup for the [MorphoTokenEthereum](../src/MorphoTokenEthereum.sol) and  [MorphoTokenOptimism](../src/MorphoTokenOptimism.sol) contracts.
+
+## Getting started
+
+This project depends on [Solidity](https://soliditylang.org/) which is required for running the verification.
+The compiler binary should be available in the path:
+
+- `solc-0.8.27` for the solidity compiler version `0.8.27`.
+
+To verify a specification, run the command `certoraRun Spec.conf` where `Spec.conf` is the configuration file of the matching CVL specification.
+Configuration files are available in [`certora/confs`](confs).
+Please ensure that `CERTORAKEY` is set up in your environment.
+
+## Overview
+
+These Morpho token contracts replace the legacy version and add support for delegation of voting power, upgradeability and cross-chain interactions.
+Due to the upgradeable nature of the new Morpho token, the contract isn't properly immutable.
+Instead, we verify that the implementation doesn't perform delegate calls.
+
+Note: the compiled contracts may include loops related to handling strings from the EIP712, for this reason the verification is carried with the option `optimistic_loop` set to `true` in order to avoid related counterexamples.
+
+### External calls
+
+This is checked in [`ExternalCalls.spec`](specs/ExternalCalls.spec).
```

Comment:
unclear from here what checking "external calls" means

Reply:
With @QGarchery, we thought that it could be in-between to avoid confusing with immutability of the Morpho token, see https://github.com/morpho-org/morpho-token-upgradeable/pull/90#discussion_r1836497340. Let me know which name to pick and I'll do a final change as needed.

Reply:
ok fine like that